llvm/lib/Support/CommandLine.cpp

```diff
@@ -2453,9 +2468,14 @@ class HelpPrinter {
     sortSubCommands(GlobalParser->RegisteredSubCommands, Subs);
 
 #ifdef SYCLomatic_CUSTOMIZATION
-    if (helpCatEnum == HelpCategory::HC_Examples) {
-      outs() << DPCTExamplesMsg;
-      return;
+    isDPCT = !strcmp((GlobalParser->ProgramName).c_str(), "dpct") ||
+             !strcmp((GlobalParser->ProgramName).c_str(), "c2s");
```

Comment:
We can define `IsDPCT` as a global variable with init value `false` in this file. And set its value to true in `runDPCT` function. Is it better?